File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Option.java
package com.bnuz.oke.entity;

/**
 * Option
 *
 * @author handsome
 * @date 2020年 05月15日 00:10:58
 */
public class Option {

	private int optionId;

	private Question question;

	private String optionType;

	private String optionDescribe;

	public int getOptionId() {
		return optionId;
	}

	public void setOptionId(int optionId) {
		this.optionId = optionId;
	}

	public Question getQuestion() {
		return question;
	}

	public void setQuestion(Question question) {
		this.question = question;
	}

	public String getOptionType() {
		return optionType;
	}

	public void setOptionType(String optionType) {
		this.optionType = optionType;
	}

	public String getOptionDescribe() {
		return optionDescribe;
	}

	public void setOptionDescribe(String optionDescribe) {
		this.optionDescribe = optionDescribe;
	}

	@Override
	public String toString() {
		return "Option{" +
				"optionId=" + optionId +
				", question=" + question +
				", optionType='" + optionType + '\'' +
				", optionDescribe='" + optionDescribe + '\'' +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/Option.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 问题选项信息封装类
    /// </summary>
    [Serializable]
    public class Option
    {
        //选项id
        public int optionId { set; get; }
        //选项所属的问题
        public Question question { set; get; }
        //选项类型
        public string optionType { set; get; }
        //选项描述
        public string optionDescribe { set; get; }
    }
}
